src/Amazon.Common.DotNetCli.Tools/Utilities.cs

```diff
@@ -205,72 +206,129 @@ public static string DeterminePublishLocation(string workingDirectory, string pr
             return path;
         }
 
-        public static string LookupTargetFrameworkFromProjectFile(string projectLocation)
-        {
-            var projectFile = FindProjectFileInDirectory(projectLocation);
 
-            var xdoc = XDocument.Load(projectFile);
-
-            var element = xdoc.XPathSelectElement("//PropertyGroup/TargetFramework");
-            return element?.Value;
-        }
-
-        /// <summary>
-        /// Retrieve the `OutputType` property of a given project
+        // <summary>
+        /// Looks up specified properties from a project file.
```

Comment:
nit: drop "file" since MSBUILD is not looking at the project file (.csproj) alone